test/PerformanceTests/SerializationComparisonsTests/README.md

```diff
@@ -0,0 +1,352 @@
+| Max Working Set (MB)                    | 194           |
+| Max GC Heap Size (MB)                   | 102           |
+| Size of committed memory by the GC (MB) | 123           |
+| Max Number of Gen 0 GCs / sec           | 64.00         |
+| Max Number of Gen 1 GCs / sec           | 18.00         |
+| Max Number of Gen 2 GCs / sec           | 2.00          |
+| Max Time in GC (%)                      | 24.00         |
+| Max Gen 0 Size (B)                      | 16,073,800    |
+| Max Gen 1 Size (B)                      | 4,861,864     |
+| Max Gen 2 Size (B)                      | 17,368,952    |
+| Max LOH Size (B)                        | 4,418,008     |
+| Max POH Size (B)                        | 1,754,688     |
+| Max Allocation Rate (B/sec)             | 5,747,380,304 |
+| Max GC Heap Fragmentation               | 54            |
+| # of Assemblies Loaded                  | 120           |
+| Max Exceptions (#/s)                    | 0             |
+| Max Lock Contention (#/s)               | 36            |
+| Max ThreadPool Threads Count            | 36            |
+| Max ThreadPool Queue Length             | 77            |
+| Max ThreadPool Items (#/s)              | 19,879        |
+| Max Active Timers                       | 0             |
+| IL Jitted (B)                           | 382,853       |
+| Methods Jitted                          | 5,311         |
+
+
+| load                   |         |
+| ---------------------- | ------- |
+| CPU Usage (%)          | 13      |
+| Cores usage (%)        | 155     |
+| Working Set (MB)       | 41      |
+| Private Memory (MB)    | 110     |
+| Start Time (ms)        | 91      |
+| First Request (ms)     | 184     |
+| Requests               | 252,387 |
+| Bad responses          | 0       |
+| Latency 50th (ms)      | 14.83   |
+| Latency 75th (ms)      | 16.16   |
+| Latency 90th (ms)      | 18.15   |
+| Latency 95th (ms)      | 19.70   |
+| Latency 99th (ms)      | 23.36   |
+| Mean latency (ms)      | 15.21   |
+| Max latency (ms)       | 161.76  |
+| Requests/sec           | 8,412   |
+| Requests/sec (max)     | 12,376  |
+| Read throughput (MB/s) | 118.45  |
+
+It's always important to look at the `Bad responses` metric before making any conclusions from the other stats.
+If Bad responses is 0, it means all requests were successful (i.e. 2xx response codes). If it's non-zero, it means there were failed requests. Common causes of failed requests include passing an unknown writer to the `--variable writer=...` variable and flooding the server with more requests than it can handle. To mitigate against the latter, you can try to limit the load on the server by setting limit the number of concurrent connections and/or the max request rate (number of requests per second). Different servers can handle different capacities, so may need to play around with different values until you find the highest load that results in 0 bad responses.

```

Comment:
```suggestion
If Bad responses is 0, it means all requests were successful (i.e. 2xx response codes). If it's non-zero, it means there were failed requests. Common causes of failed requests include passing an unknown writer to the `--variable writer=...` variable and flooding the server with more requests than it can handle. To mitigate against the latter, you can try to limit the load on the server by limiting the number of concurrent connections and/or the max request rate (number of requests per second). Different servers can handle different capacities, so may need to play around with different values until you find the highest load that results in 0 bad responses.
```